### PR TITLE
My profile

### PR DESCRIPTION
**Pull request by Kwolik**

### CODE REVIEW

File: web-client/src/components/ChangePassword.jsx [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+function ChangePassword() {
+  const styles = makeStyles({
+    main: {
+      padding: "2%",
+      display: "flex",
+    },
+    // columns:
+    // {
+    //     display: "flex",
+    // },
```
Comment (upsu90) [inline web-client/src/components/ChangePassword.jsx]: @JustD4nTe will be angry :c

File: web-client/src/components/PersonalInformation.jsx [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+                label="I accept the Rules of Scienture Conference and I agree to processing my personal data included in the above form by...*"
+                inputRef={register}
+                name="acceptance"
+              />
+            :null}
+            {!props.firstName && !props.lastName && !props.email ? 
+            <FormHelperText error className={style.formHelperText}>
+              {errors.acceptance ? errors.acceptance.message : " "}
+            </FormHelperText>
+            : null}
```
Comment (upsu90) [inline web-client/src/components/PersonalInformation.jsx]: we don't need acceptance again

**Review by upsu90 — CHANGES_REQUESTED** [review]

File: web-client/src/components/PersonalInformation.jsx [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+            {props.participant ? <Avatar name='avatar'/> : null}
+
+            {/* Button Submit */}
+            <Button
+              className={style.btnSignup}
+              color="primary"
+              type="submit"
+              variant="contained"
+            >
+              {!props.firstName && !props.lastName && !props.email ? "Sign up" : "Save changes"}
```
Comment (upsu90) [inline web-client/src/components/PersonalInformation.jsx]: Do we need "Sign up" here?

**Review by upsu90 — CHANGES_REQUESTED** [review]

**Review by upsu90 — APPROVED** [review]

**Review by KamilDonda — APPROVED** [review]

File: web-client/src/components/ChangePassword.jsx [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+
+    SetOpenAlertSuccess(false);
+    SetOpenAlertError(false);
+  };
+
+  // Current Password
+  const [currentPassword, SetCurrentPassword] = React.useState({
+    password: "",
+    showPassword: false,
+  });
```
Comment (JustD4nTe) [inline web-client/src/components/ChangePassword.jsx:102]: Pointless comments

File: web-client/src/components/ChangePassword.jsx [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+
+  const handleMouseDownCurrentPassword = (event) => {
+    event.preventDefault();
+  };
+
+  // New Password
+  const [newPassword, SetNewPassword] = React.useState({
+    password: "",
+    showPassword: false,
+  });
```
Comment (JustD4nTe) [inline web-client/src/components/ChangePassword.jsx:123]: adn here too

**Review by JustD4nTe — APPROVED** [review]

**Pull request merged**